src/client/src/index.tsx

```diff
@@ -1,12 +1,18 @@
-import 'rt-theme'
+import React from 'react'
+import ReactDOM from 'react-dom'
+import { BrowserRouter, Route, Switch } from 'react-router-dom'
 
-import { parse } from 'query-string'
-import { run } from './notificationBootstrapper'
+import 'rt-theme'
 
-const parsed = parse(location.search)
+import MainRoute from './MainRoute'
+import NotificationRoute from './NotificationRoute'
 
-if (parsed.notification) {
-  run()
-} else {
-  import('./main').then(mod => mod.run())
-}
+ReactDOM.render(
+  <BrowserRouter>
+    <Switch>
+      <Route path="/notification" component={NotificationRoute} />
```

Comment:
Imports are now without side effect.

We can add `react-loadable` if we can show value for async splitting — I could not find significant value when examined. The bundle will be a cache read for electron/chrome.

I could see value once styleguide is introduced as it may entail a number of additional content assets — though the effect will likely be under 25kb, which remains insignificant.

Reply:
Just checked. It is cached

Reply:
Nice, thanks for verifying!